dissect/target/loaders/ab.py

```diff
@@ -21,7 +23,10 @@
 from dissect.target.helpers import keychain
 from dissect.target.loader import Loader
 from dissect.target.plugins.os.unix.linux.android._os import AndroidPlugin
-from dissect.target.target import Target
+
+if TYPE_CHECKING:
+    from dissect.target import Target
```

Comment:
```suggestion
    from dissect.target.target import Target
```
I'd make it the full import everywhere.